packages/preview/src/browser/preview-contribution.ts

```diff
@@ -22,7 +22,7 @@ import { PreviewUri } from "./preview-uri";
 export namespace PreviewCommands {
     export const OPEN: Command = {
         id: 'preview:open',
-        label: 'Open Preview'
+        label: 'Preview: Open'
```

Comment:
Maybe with 'Open' we should use the form `Open <What>`

Reply:
Yea that's what I was also thinking, this could be added to the documentation also, for `Open ...` stuff it's better.